Makefile

```diff
@@ -55,12 +55,21 @@ install-travis:
 # Example:
 #   make check
 #   make test
-#   make check WHAT=pkg/build TESTFLAGS=-v
-check:
+#   make check WHAT=pkg/docker TESTFLAGS=-v
+check:	verify
```

Comment:
i don't think there is one, since you're not passing any arguments, thus there's no reason to do it that way.